playbooks/roles/caasp4/templates/skuba-terraform-libvirt.tfvars.j2

```diff
@@ -13,31 +13,34 @@ masters = {{ master_count }}
 workers = {{ worker_count }}
 
 repositories = {
-  caasp_devel = "{{ ibs_mirror_uri }}/Devel:/CaaSP:/4.0/SLE_15_SP1/"
-  suse_ca = "{{ ibs_mirror_uri }}/SUSE:/CA/SLE_15_SP1/"
-  sle_server_pool = "{{ rmt_mirror_uri }}/SUSE/Products/SLE-Product-SLES/15-SP1/x86_64/product/"
-  basesystem_pool = "{{ rmt_mirror_uri }}/SUSE/Products/SLE-Module-Basesystem/15-SP1/x86_64/product/"
-  containers_pool = "{{ rmt_mirror_uri }}/SUSE/Products/SLE-Module-Containers/15-SP1/x86_64/product/"
-  serverapps_pool = "{{ rmt_mirror_uri }}/SUSE/Products/SLE-Module-Server-Applications/15-SP1/x86_64/product/"
-  sle_server_updates = "{{ rmt_mirror_uri }}/SUSE/Updates/SLE-Product-SLES/15-SP1/x86_64/update/"
-  basesystem_updates = "{{ rmt_mirror_uri }}/SUSE/Updates/SLE-Module-Basesystem/15-SP1/x86_64/update/"
-  containers_updates = "{{ rmt_mirror_uri }}/SUSE/Updates/SLE-Module-Containers/15-SP1/x86_64/update/"
-  serverapps_updates = "{{ rmt_mirror_uri }}/SUSE/Updates/SLE-Module-Server-Applications/15-SP1/x86_64/update/"
+  caasp_pool = "{{ ibs_mirror }}/SUSE/Products/SUSE-CAASP/4.0/x86_64/product/"
+  sle_server_pool = "{{ ibs_mirror }}/SUSE/Products/SLE-Product-SLES/15-SP1/x86_64/product/"
+  basesystem_pool = "{{ ibs_mirror }}/SUSE/Products/SLE-Module-Basesystem/15-SP1/x86_64/product/"
+  containers_pool = "{{ ibs_mirror }}/SUSE/Products/SLE-Module-Containers/15-SP1/x86_64/product/"
+  serverapps_pool = "{{ ibs_mirror }}/SUSE/Products/SLE-Module-Server-Applications/15-SP1/x86_64/product/"
+  caasp_updates = "{{ ibs_mirror }}/SUSE/Updates/SUSE-CAASP/4.0/x86_64/update/"
+  sle_server_updates = "{{ ibs_mirror }}/SUSE/Updates/SLE-Product-SLES/15-SP1/x86_64/update/"
+  basesystem_updates = "{{ ibs_mirror }}/SUSE/Updates/SLE-Module-Basesystem/15-SP1/x86_64/update/"
+  containers_updates = "{{ ibs_mirror }}/SUSE/Updates/SLE-Module-Containers/15-SP1/x86_64/update/"
+  serverapps_updates = "{{ ibs_mirror }}/SUSE/Updates/SLE-Module-Server-Applications/15-SP1/x86_64/update/"
```

Comment:
In general, I like this change as it makes sense. 
In my local setup, I encountered frequent timeout during cloud-init of nodes and primarily issue was related to kernal-default package download and install which is around 250 MB.
So to get around this issue, I did end up setting up RMT server locally and then set up basesystem_pool and baseystem_updates referencing from that local RMT repos. Having this name as RMT server uri primarily gave the idea otherwise I even may not have looked into this approach.
Point is that in situations like this, having RMT server uri different from ibs mirror can be useful.

Reply:
> Point is that in situations like this, having RMT server uri different from ibs mirror can be useful.

I am non entirely sure what you mean here. As far as I can see all repos that use `ibs_mirror` in socok8s currently can be mirrored using an RMT. That's why I remove the the `rmt_mirror` thing.

Reply:
ibs_mirror is used throughout in different plays and can have many more repos which are not needed in cloud-init steps. So if rmt and ibs mirror URI are same that means RMT server needs to have all those additional repos mirrored as well. Issue may not appear important unless the setup is done on slow VPN.

Anyway, that's more of workaround and specific to env, so we may not have to handle it here. So fine with ibs_mirror if that's the preference.